.gitignore

```diff
@@ -5,7 +5,8 @@
 db.json
 dist/
 node_modules/
-src/hexo/source/about/
-src/hexo/source/docs/contributor-guide
-src/hexo/source/docs/user-guide
+src/content/about/
+src/content/docs/contributor-guide/
```

Comment:
A lot of `src/content/docs/*` files are also committed. Is that right?

Reply:
I think they are leftovers from before that were commited before being ignored in `src/hexo/source/`,
In this case, I just moved them and add the entry to `.gitignore` so git has probably pick it up as a `mv`.

I'll see if I can remove them.

Reply:
This should be fixed